cdap-master/src/main/java/co/cask/cdap/data/tools/flow/FlowQueuePendingCorrector.java

```diff
@@ -354,30 +351,13 @@ protected void configure() {
           bind(QueueClientFactory.class).to(HBaseQueueClientFactory.class).in(Singleton.class);
           bind(QueueAdmin.class).to(HBaseQueueAdmin.class).in(Singleton.class);
           bind(HBaseTableUtil.class).toProvider(HBaseTableUtilFactory.class);
-        }
-
-        @Provides
-        @Singleton
-        @Named("defaultStore")
-        @SuppressWarnings("unused")
-        public Store getStore(CConfiguration conf,
-                              LocationFactory locationFactory,
-                              NamespacedLocationFactory namespacedLocationFactory,
-                              final TransactionExecutorFactory txExecutorFactory,
-                              DatasetFramework framework,
-                              TransactionSystemClient txClient) {
-          return new DefaultStore(conf, locationFactory, namespacedLocationFactory,
-                                  txExecutorFactory, framework, txClient);
-        }
+          bind(Store.class).annotatedWith(Names.named("defaultStore")).to(DefaultStore.class).in(Singleton.class);
```

Comment:
I thought about it, but that would involve introducing a common parent as the Tool base, which is a bigger change than the intention of this PR (which is to cleanup the master process guice modules).